test/test_device.py

```diff
@@ -215,15 +215,53 @@ def _test_signtx(self, input_type):
         pkh_addr = self.wrpc.getnewaddress('', 'legacy')
         self.wrpc.importaddress(wpkh_addr)
         self.wrpc.importaddress(pkh_addr)
-        send_amount = 10
+
+        # pubkeys to construct 2-of-3 multisig descriptors for import
+        sh_wpkh_info = self.wrpc.getaddressinfo(sh_wpkh_addr)
+        wpkh_info = self.wrpc.getaddressinfo(wpkh_addr)
+        pkh_info = self.wrpc.getaddressinfo(pkh_addr)
+
+        # Get origin info/key pair so wallet doesn't forget how to
+        # sign with keys post-import
+        pubkeys = [sh_wpkh_info['desc'][8:-2],\
+                wpkh_info['desc'][5:-1],\
+                pkh_info['desc'][4:-1]]
+
+        sh_multi_desc = [{'desc':'sh(multi(2,'+pubkeys[0]+','+pubkeys[1]+','+pubkeys[2]+'))', "timestamp":"now", "label":"shmulti"}]
+        sh_wsh_multi_desc = [{'desc':'sh(wsh(multi(2,'+pubkeys[0]+','+pubkeys[1]+','+pubkeys[2]+')))', "timestamp":"now", "label":"shwshmulti"}]
+        # re-order pubkeys to allow import without "already have private keys" error
+        wsh_multi_desc = [{'desc':'wsh(multi(2,'+pubkeys[2]+','+pubkeys[1]+','+pubkeys[0]+'))', "timestamp":"now", "label":"wshmulti"}]
+        sh_multi_result = self.wrpc.importmulti(sh_multi_desc)
+        sh_wsh_multi_result = self.wrpc.importmulti(sh_wsh_multi_desc)
+        wsh_multi_result = self.wrpc.importmulti(wsh_multi_desc)
```

Comment:
Instead of calling `importmulti` 3 times, can you have it call `importmulti` once and import all three descriptors at the same time?

Reply:
galaxy brain, done

Reply:
galaxy brain, done